tools/validators/instance_validator/validate/entity_instance.py

```diff
@@ -114,11 +115,16 @@ def _ParseTranslation(self, translation_body):
 
     translation = {}
 
-    for field_name in translation_body.keys():
-      if isinstance(translation_body[field_name], str):
+    for std_field_name in translation_body.keys():
+      if isinstance(translation_body[std_field_name], str):
         continue
 
-      ft = translation_body[field_name]
+      ft = translation_body[std_field_name]
+
+      raw_field_name = str(ft[PRESENT_VALUE])\
```

Comment:
I don't follow this at all.  Explain what's happening in a comment plz?

Reply:
this part extracts the foo part in the present_value: "points.foo.present_value"

```
CHWS_WDT-17:
  type: HVAC/CHWS_WDT
  id: CDM/123456
  translation:
    return_water_temperature_sensor:
      present_value: "points.return_water_temperature_sensor.present_value"
      ```

Reply:
why is foo important?  The payload could have any arbitrary json path as the present_value and still be valid.

Reply:
Discussed offline with Keith, created the following b/176097512